pkg/controller/migration/convert/core.go

```diff
@@ -156,6 +156,14 @@ func handleCore(c *components, install *Installation) error {
 		return err
 	}
 
+	epHostAction, err := c.node.getEnv(ctx, c.client, containerCalicoNode, "FELIX_DEFAULTENDPOINTHOSTACTION")
```

Comment:
This check is already in network.go. You should remove it from there.
I think having it here is probably the right place.

Reply:
I thought the same thing, but I think we both were confusing it with `FELIX_IPTABLESMANGLEALLOWACTION` because I checked and there are no checks for `FELIX_IPTABLESMANGLEALLOWACTION`.

Reply:
https://github.com/tigera/operator/blob/tech-preview-migrate-install/pkg/controller/migration/convert/network.go#L22 is what I'm talking about

Reply:
Whoops! This one had a typo in the env var name which is why I didn't find the other. Fixed!